cypress/integration/workflows_spec.js

```diff
@@ -52,5 +55,15 @@ describe('Dashboard Workflows Page', () => {
           .to.deep.equal({Type: 'Task', Resource: '${SfSnsReportLambdaAliasOutput}', Next: 'StopStatus'});
       });
     });
+
+    it('filters workflows when a user types in the search box', () => {
+      cy.server();
+      cy.route('GET', '/workflows*').as('get-workflows');
+      cy.visit('/workflows');
+      cy.get('.table .tbody .tr').its('length').should('be.eq', 2);
+      cy.get('.table .tbody .tr').first().contains('HelloWorldWorkflow');
+      cy.get('.search').click().type('cond');
```

Comment:
love that this can be in the middle of the name instead of starting at the beginning

Reply:
It is the middle.  the string is "SecondTestWorkflow", so 'cond', but I will change it to 
'conndtes' to make sure it's case insensitve.

Reply:
7ef28d16